v1/anbox-cloud-appliance.yaml

```
@@ -0,0 +1,21 @@
+description: Anbox Cloud Appliance
+version: latest
+
+instances:
+  anbox-cloud-appliance:
+    limits:
+      min-cpu: 4
+      min-mem: 4G
+      min-disk: 50G
+    timeout: 900
+    cloud-init:
+      vendor-data: |
+        runcmd:
+        - apt update
+        - DEBIAN_FRONTEND=noninteractive apt purge -y linux-kvm linux-image-kvm linux-headers-kvm linux-image-* linux-headers-* linux-modules-*
```

This should do, then?
```suggestion
        - DEBIAN_FRONTEND=noninteractive apt purge -y linux-image-* linux-headers-* linux-modules-*
```